source/as-stream.js

```diff
@@ -4,47 +4,49 @@ const duplexer3 = require('duplexer3');
 const is = require('@sindresorhus/is');
 const requestAsEventEmitter = require('./request-as-event-emitter');
 const {HTTPError, ReadError, RequestError} = require('./errors');
+const normalizeArguments = require('./normalize-arguments');
 
-module.exports = options => {
-	options.stream = true;
+module.exports = (url, options) => {
+	const normalizedArgs = normalizeArguments(url, options);
+	normalizedArgs.stream = true;
```

Comment:
`normalizedArgs` => `normalizedOptions` would be more accurate, but that's quite verbose.

What's wrong with just:

```js
options = normalizeArguments(url, options);
```

Do we need the original options somewhere?

Reply:
Fixed that.